$OpenBSD: patch-ktorrent_dialogs_torrentcreatordlg_h,v 1.1 2018/05/29 21:01:44 rsadowski Exp $

"Some generated files have includes in a different order than in older
CMake versions, which show up missing typedefs / implicit includes."

From FreeBSD


Index: ktorrent/dialogs/torrentcreatordlg.h
--- ktorrent/dialogs/torrentcreatordlg.h.orig
+++ ktorrent/dialogs/torrentcreatordlg.h
@@ -23,6 +23,7 @@
 #define KT_TORRENTCREATORDLG_HH
 
 #include <QDialog>
+#include <QTimer>
 #include "ui_torrentcreatordlg.h"
 #include <torrent/torrentcreator.h>
 
